### PR TITLE
 update overlaytrigger and tooltip with withtooltip

### PR DESCRIPTION
**Pull request by pathakyashwardhan**
#### Release Note

```release-note
release-note-none
```

### CODE REVIEW

File: webapp/channels/src/components/custom_status/custom_status_suggestion.tsx [hunk 1]
```diff
@@ -9,8 +9,7 @@ import type {UserCustomStatus} from '@mattermost/types/users';
 import {CustomStatusDuration} from '@mattermost/types/users';
 
 import RenderEmoji from 'components/emoji/render_emoji';
-import OverlayTrigger from 'components/overlay_trigger';
-import Tooltip from 'components/tooltip';
+import WithTooltip from 'components/with_tooltip'; //Import the WithTooltip component
```
Comment (larkox) [inline webapp/channels/src/components/custom_status/custom_status_suggestion.tsx:12]: The comment is not needed.

File: webapp/channels/src/components/custom_status/custom_status_suggestion.tsx [hunk 2]
```diff
@@ -42,25 +41,19 @@ const CustomStatusSuggestion: React.FC<Props> = (props: Props) => {
...
+        <div className='suggestion-clear'>
+            <WithTooltip
                 delayShow={Constants.OVERLAY_TIME_DELAY}
                 placement='top'
-                overlay={
-                    <Tooltip id='clear-recent-custom-status'>
-                        {'Clear'}
-                    </Tooltip>
-                }
+                tooltipText={'Clear'}
```
Comment (larkox) [inline webapp/channels/src/components/custom_status/custom_status_suggestion.tsx:48]: Out of the scope, but can we translate this string? instantiate `intl` by doing `const intl = useIntl();` and then here something like:

```suggestion
                tooltipText={intl.formateMessage({id: 'input.clear', defaultMessage: 'Clear'})}
```

**Review by larkox — CHANGES_REQUESTED** [review]
Great work! A couple of small improvements.

File: webapp/channels/src/components/custom_status/custom_status_suggestion.tsx (hunk 1 above)
Comment (pathakyashwardhan) [reply webapp/channels/src/components/custom_status/custom_status_suggestion.tsx]: All changes are done

**Comment by pathakyashwardhan** [comment]
all changes are updated

**Comment by M-ZubairAhmed** [comment]
@pathakyashwardhan Ah there was an already existing PR to this issue which got merged already. I know this is not the experience you would want to have from the project and we are sorry about that but we have to close this one out. There are more open issues related to the above campaign. Can you head over to https://github.com/mattermost/mattermost/issues?q=is%3Aissue+is%3Aopen+to+WithTooltip+of+label%3A%22Help+Wanted%22+ to take a look ?

**Pull request closed without merging**